cockroachcloud/upgrade-to-v20.2.md

```diff
@@ -55,7 +55,11 @@ Review the [backward-incompatible changes in v20.2](../releases/v20.2.0.html#bac
 
 Make sure there are no [bulk imports](../stable/import.html) or [schema changes](../stable/online-schema-changes.html) in progress. These are complex operations that can increase the potential for unexpected behavior during an upgrade.</span>
 
-To check for ongoing bulk operations, use [`SHOW JOBS`](https://www.cockroachlabs.com/docs/v20.1/show-jobs.html#show-schema-changes) or check the [**Jobs** page](../stable/admin-ui-jobs-page.html) in the Admin UI.
+To check for ongoing bulk operations, use [`SHOW JOBS`](https://www.cockroachlabs.com/docs/stable/show-jobs.html#show-schema-changes) or check the [**Jobs** page](../stable/admin-ui-jobs-page.html) in the Admin UI.
+
+{{site.data.alerts.callout_danger}}
```

Comment:
@lucy-zhang, PTAL. Is this ok, or should we keep it even simpler and just mentioned that buggy schema changes that are not in a terminal state will just be marked `failed` during the upgrade to 20.2, as you suggest here: https://github.com/cockroachdb/cockroach/pull/54902#issuecomment-700406271 ?